.vib/flux/goss/goss-wait.yaml

```diff
@@ -9,3 +10,13 @@ addr:
   tcp://podinfo:9898:
     reachable: true
     timeout: 2000
```

Comment:
Is `podinfo` available in an `air-gapped` environments as well? If that is the case, we can leave this one for `air-gapped` and leave the other related to the chart for other environments

Reply:
it's a pod deployed via a repo that is cloned and the clone doesn't work in air-gapped envs

https://github.com/stefanprodan/podinfo/tree/master/kustomize